src/NzbDrone.Core/MediaFiles/EpisodeImport/DetectSample.cs

```diff
@@ -43,6 +43,19 @@ public DetectSampleResult IsSample(Series series, string path, bool isSpecial)
                 _logger.Debug("Skipping sample check for .strm file");
                 return DetectSampleResult.NotSample;
             }
+            
+            string[] videoExtensions = {".webm", ".mkv", ".flv", ".mts", ".m2ts", ".mt", ".mov", ".wmv", ".amv", ".mp4", ".mpg", ".mpeg", ".mpv", ".m4v", "3gp", "3g2", ".flv", ".avi"};
+            
+            bool isVideo = false;
+            for (int i = 0; i < videoExtensions.Length; i++){
+            	if (extension != null && extension.Equals(videoExtensions[i], StringComparison.InvariantCultureIgnoreCase)){
+               	isVideo = true;
+                	break;
+            	}
+	    }
+	    if (!isVideo){
```

Comment:
Most users do not run with debug nor would see the debug message.

Reply:
I made the pull request after a heated argument and i will be closing it. Sorry.